### DISCUSSION TITLE
DNG Tagging (or other raw formats)

### DISCUSSION THREAD

**Discussion by artisak2** [deckerst/aves#952] [Ideas]
Hi,
I was wondering if it is possible to add tags to raw formats such as .dng?
I do not have access to Lightroom at the moment, but people claim that it can embed tags/keywords into .dng files (no .xmp sidecar)

If possible, is this something that may be of interest to implement?

Thanks for the wonderful app!

**Comment by deckerst** [comment]
Not possible at the moment. See [the wiki](https://github.com/deckerst/aves/wiki/Support:-Metadata-editing) for current support.

Ideally, I'd like to support metadata editing for DNG and HEIC (because mobile cameras often produce these), but I do not implement that support myself. I depend on libraries, and they do not support these formats (yet).

As for other raw formats, it probably will never be supported.

**Reply by artisak2** [reply]
That is understandable.

I am trying to come up with an organization workflow that would include dng's. Based on your response it seems like the only other option may be to tag files via filename.

I wonder if it would make sense to add a rename feature that can pull metadata tags from the file in a batch rename operation? (or any other metadata information, for that matter)
   (Like the app can currently for Date/Time via <date, yyyyMMdd-HHmmss> )

**Reply by deckerst** [reply]
I'm interested in providing something like this but I haven't looked into it. Please let me know in practice what field/tag/modifiers you need. For example, "i need the image width, the Exif GPS altitude, and the XMP Dublin Core subject list".

**Reply by artisak2** [reply]
I would need XMP Dublin Core subject list.

I would also find use of EXIF Make & Model.

From a "power" users perspective it is always good to have more options available. But at the moment, i would only see myself using the ones above.

**Reply by deckerst** [reply]
Done!

**Reply by artisak2** [reply]
Awesome.
I will be awaiting the next release.
